src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const allMen = people.filter(item => item.sex === 'm');
+  const menForNeedCentury = allMen.filter(person =>
+    Math.ceil(person.died / 100) === century);
+  const countAges = menForNeedCentury.length;
+  const countAllAges = allMen.length;
```

Comment:
These three constants could be removed, and allMen array can already contain filtered values by right condition (and renamed)